### PR TITLE
add script to delte all completed tasks

### PR DESCRIPTION
**Pull request by MAHDIA001**
in this milestone, I have created new files to add functionalities to the website.

### CODE REVIEW

File: src/complete.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Storage from './storage.js';
+
+const tasksArray = JSON.parse(localStorage.getItem('todo')) || [];
+const complete = () => {
+  const check = document.querySelectorAll('input[type=checkbox]');
+  check.forEach((input, index) => input.addEventListener('change', () => {
+    if (input.checked) {
+      tasksArray[index].completed = true;
```
Comment (ha-manel) [inline src/complete.js]: - [x] When we click on the checkbox kindly add a line through the checked item.

File: src/main.js [hunk 2]
```diff
@@ -63,5 +63,33 @@ toDoInput.addEventListener('keypress', (e) => {
...
+      //    list[index].style.textDecoration = 'none';
+      //    list[index].style.color = 'black';
+    }
+    Storage(tasksArray);
+  }));
+};
+
+const clear = document.querySelector('#clear-button');
+clear.addEventListener('click', () => {
+  const completed = tasksArray.filter((data) => data.completed === true);
```
Comment (ha-manel) [inline src/main.js]: - [ ] The `clear completed` functionality isn't working properly, kindly fix this issue 👍 

Animation

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @MAHDIA001 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.js (hunk 2 above)
Comment (MAHDIA001) [reply src/main.js]: hello manel 
can you please help me out with how to fix it?

File: src/complete.js (hunk 1 above)
Comment (MAHDIA001) [reply src/complete.js]: this is optional not required

**Review by emmiiorji — APPROVED** [review]
Hi @MAHDIA001, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Linter checks are passing
- Adherence to GitHub flow
- Good commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**